pkg/cmd/server/start/start_master.go

```diff
@@ -531,6 +533,8 @@ func StartAPI(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) error {
 	// start up the informers that we're trying to use in the API server
 	oc.Informers.InternalKubernetesInformers().Start(utilwait.NeverStop)
 	oc.Informers.Start(utilwait.NeverStop)
+	// need to start route informer for public pull urls (image strategy)
+	go oc.RouteInformers.Route().InternalVersion().Routes().Informer().Run(utilwait.NeverStop)
```

Comment:
@smarterclayton if we are going to create a controller that will set this field automatically for existing image streams we will need to run informer for routes I guess.